react/components/EditorProvider.tsx

```diff
@@ -31,6 +31,24 @@ interface State {
 // tslint:disable-next-line:no-empty
 const noop = () => {}
 
+const getUrlProperties = (href: string) => {
+  const match: string = href.match(
```

Comment:
`match` can return `string | null`, so this raises a ts error.
You don't have to explicitly declare match type, since `href.match(..)` will implicitly define it.
But if you do, define it as `string | null`